slices/libgcc-s1.yaml

```diff
@@ -4,5 +4,12 @@ slices:
   libs:
     essential:
       - libc6_libs
+      - libgcc-s1_copyright
     contents:
       /lib/*-linux-*/libgcc_s.so.*:
+
+  copyright:
+    essential:
+      - gcc-12-base_copyright
```

Comment:
Can you please also add these copyright essentials to the top level essentials?
```yaml
package: libgcc-s1

essential:
    - libgcc-s1_copyright

slices:
  libs:
    essential:
      - libc6_libs
    contents:
      /lib/*-linux-*/libgcc_s.so.*:

  copyright:
    essential:
      - gcc-12-base_copyright
    contents:
      /usr/share/doc/libgcc-s1:
```

and same for the other slice.